devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy

```diff
@@ -146,6 +146,13 @@ def zwaveEvent(physicalgraph.zwave.commands.alarmv2.AlarmReport cmd, results) {
 			if(device.currentValue("smoke") != "clear") {
 				createSmokeOrCOEvents("smokeClear", results)
 			}
+
+			if (device.currentValue("carbonMonoxide") != "clear") {
+				def zwMap = getZwaveInfo()
+				if (zwMap && zwMap.mfr == "0138" && zwMap.prod == "0001" && zwMap.model == "0002") {
+					createSmokeOrCOEvents("carbonMonoxideClear", results)
+				}
```

Comment:
Well, in this case, this particular message from the detector is an "all clear here" message. So, if smoke or carbonMonoxide are not marked as clear then we should.